examples/static-react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "vip-go-node-skeleton-examples-static-react",
+  "version": "0.0.1",
+  "description": "Node static service with React example for WPVIP",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "npx @automattic/vip-go-preflight-checks --verbose --wait=2000",
+    "build": "cd client && npm install && npm run build",
```

Comment:
Hmmm I missed this on my first pass. Most customer examples I've seen don't have a nested project like this, and I can foresee some resistance to that. A single `package.json` and server code in a `server` directory is a more common pattern (and one hinted at in the `create-react-app` docs):

https://create-react-app.dev/docs/deployment/#static-server

Reply:
Indeed, that's another option - modify the client project to add the server - that might be more confusing to some customers (and it'd be where they forget to add the healthcheck route). So I think it's a bit more advanced than the concepts I wanted to convey here.

I think we can - and should - discuss that option, in the documentation, but I wanted to make this closer to the simple example, and allow for the situation where folks are reticent to make many changes to their React project, or reticent to add unrelated code, even if it's for serving; they may be worried it'd get into the build somehow. 

I'm also considering the fact some customers might have many other bits of server logic to add (which we can demonstrate with more advanced projects).

What do you think?